packages/commonwealth/server/CommonwealthConsumer/messageProcessors/snapshotConsumer.ts

```diff
@@ -77,6 +73,19 @@ export async function processSnapshotMessage(
     });
   }
 
+  // Notifications
+  emitNotifications(
+    this.models,
+    NotificationCategories.SnapshotProposal,
+    snapshotNotificationData.space,
+    { eventType, ...snapshotNotificationData },
+    {
+      notificationCategory: eventType,
+      body: snapshotNotificationData.body,
+      title: snapshotNotificationData.title, // TODO: Decide on what we want for the webhook we emit
```

Comment:
I think this is fine, bc we don't currently expose this as a webhook option in the admin modal for webhooks.